frontend/src/lib/i18n/locales/en.json

```diff
@@ -37,32 +37,32 @@
         "list_winners": "List winners",
         "get_ranking_and_winners": "Get the ranking and see who won",
         "why_classquiz": "Why ClassQuiz?",
-        "no_tracking_content": "Kahoot tracks with at least 2 american 3rd-parties and ClassQuiz doesn't track with any 3rd-parties at all!",
+        "no_tracking_content": "Kahoot! tracks you with at least two American third-parties, whereas ClassQuiz doesn't at all!",
```

Comment:
This is wrong, since it contains analytics, which are running on the same server as ClassQuiz and there is no 3rd-party associated, but it uses plausible and Sentry, so I wouldn't consider ClassQuiz as tracking-free.

Reply:
It might be a unfair comparison to even call analytics vs. whatever sending data to american third-parties is?
What is that for anyway?
I'd rather it was the point to make that there is no tracking _unless you ask it to do so_, because it is a great point to make.